metarecord/tests/test_importer.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import pytest
+
+from metarecord.importer.tos import TOSImporter, TOSImporterException
+
+EXCEL_FILENAME = "00 00 01 02.xlsx"
+CURRENT_FILE = os.path.realpath(__file__)
+CURRENT_DIR = os.path.dirname(CURRENT_FILE)
+PATH = os.path.join(CURRENT_DIR, "test_data", EXCEL_FILENAME)
+
+
+@pytest.mark.django_db
+def test_tos_importer_with_no_classification():
+    tos_importer = TOSImporter()
+    tos_importer.open(PATH)
+    try:
+        tos_importer.import_data()
+    except TOSImporterException as e:
+        assert str(e) == "Classification 00 00 01 02 does not exist"
+    else:
+        assert False, "Function save should have failed!"
+
+
+@pytest.mark.django_db
+def test_tos_importer_with_classification_and_function(function):
+    function.classification.code = "00 00 01 02"
+    function.classification.save(update_fields=["code"])
+
+    path_to_current_file = os.path.realpath(__file__)
+    current_directory = os.path.dirname(path_to_current_file)
+    path = os.path.join(current_directory, "test_data", EXCEL_FILENAME)
```

Comment:
I think this path could be a global variable in this test module, since finding the correct path to test data file isn't really part of the test.